apps/designer/app/dashboard/projects/project-card.tsx

```diff
@@ -232,6 +233,9 @@ export const ProjectCard = ({
       <DeleteProjectDialog
         isOpen={isDeleteDialogOpen}
         onOpenChange={setIsDeleteDialogOpen}
+        onDelete={() => {
+          setIsHidden(true);
```

Comment:
Should be resetted in case of data.errors ?

Reply:
yep